R/tuneParams.R

```diff
@@ -266,6 +276,28 @@ tuneParams <- function(params,
                 tipify(actionButton("next_sp", HTML("<u>n</u>ext")),
                        title = "Select next species. Keyboard shortcut: n"))
             })
+
+        output$re_sel <- renderUI({
```

Comment:
The resource selector should be part of `resourceControl`. If people don't want to control the resources then it should not appear. If you put the selector here rather than into `resourceControl` it would always show up.

Reply:
Do you mean, instead of having the resource's parameters in the left sidebar, I should move everything in the resource tab? With the resource slider and parameters at the top of the tab for example?

Reply:
No, I mean it should be in the left sidebar in the "Resource" section created by `researchControlUI()` and processed by `researchControl()`.

Reply:
I'm a bit stuck there. `resourceControlUI()` needs an input argument to know which resource parameter's should be displayed. That's the `re` that you were commenting about earlier. However if the slider that determines `re` is in `resourceControlUI()` itself, the UI won't start in the first place to determine `re`. What am I missing here? 
Or should I remove "resource" from the `controls` argument and make a small resource section below the `sp_params` UI?

Reply:
Ah, you are right. The `....ControlUI()` functions did not have access to the `input` list. I have now changed that, see https://github.com/sizespectrum/mizerExperimental/commit/58a4d0566e37a133e735bc4ac670dad0e41359ea. At the same time I got rid of the `sp` argument, because that is no longer necessary now that the functions have access to `input$sp`.

You should now add to `resourceControlUI()` a drop-down selector for the resource, setting `input$re`. When the gadget first starts and `input$re` is not yet set, you should just display the controls for the first resource.